inc/saiswitch.h

```diff
@@ -2873,6 +2879,32 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_HOSTIF_OPER_STATUS_UPDATE_MODE,
 
+    /**
+     * @brief VOQ chassis isolation setting.
+     *
+     * This attribute is for VOQ Chassis only
+     * This attribute is for configuring threshold value of active fabric ports below which VOQ Chassis isolation mechanism will trigger. Value of 0 means this attribute is not considered by switch.
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 0
+     * @validonly SAI_SWITCH_ATTR_TYPE == SAI_SWITCH_TYPE_VOQ
+     */
+    SAI_SWITCH_ATTR_FABRIC_PORT_THRESHOLD_LOW,
+
+    /**
+     * @brief VOQ chassis isolation setting.
+     *
+     * This attribute is for VOQ Chassis only
+     * This attribute is for configuring threshold value of active fabric ports above which VOQ Chassis isolation mechanism will be aborted or VOQ chassis will be eligible to become part of dis-aggregated chassis. Value of 0 means this attribute is not considered by switch.
```

Comment:
Similar to above, here, we should call out that SAI adapter triggers callback when then number of UP fabric ports exceeds this threshold and SAI adapter host callback processing should unisolate by setting to false.

This will use sai_switch_state_change_notification_fn as notification function? In that case that needs to be captured here as well?

Reply:
https://github.com/opencomputeproject/SAI/pull/1793/commits/64d77e084762eeb7ee35ac1ab1641b8584104c6d